dash/testing/dash_page.py

```diff
@@ -42,9 +42,17 @@ def window_store(self):
 
     def _wait_for_callbacks(self):
         if self.window_store:
-            return self.redux_state_rqs and all(
```

Comment:
We don't need this first `self.redux_state_rqs and` anymore?

Reply:
good catch, I was wrong about the `all`, an empty list will return True 🐛